<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncyclopediaWindow</class>
 <widget class="QMainWindow" name="EncyclopediaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Энциклопедия полезных растений</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>900</width>
     <height>700</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="aButton">
          <property name="text">
           <string>А</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bButton">
          <property name="text">
           <string>Б</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="v1Button">
          <property name="text">
           <string>В</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="gButton">
          <property name="text">
           <string>Г</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dButton">
          <property name="text">
           <string>Д</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="eButton">
          <property name="text">
           <string>Е</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="jButton">
          <property name="text">
           <string>Ж</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zButton">
          <property name="text">
           <string>З</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="iButton">
          <property name="text">
           <string>И</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="kButton">
          <property name="text">
           <string>К</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lButton">
          <property name="text">
           <string>Л</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mButton">
          <property name="text">
           <string>М</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nButton">
          <property name="text">
           <string>Н</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="oButton">
          <property name="text">
           <string>О</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pButton">
          <property name="text">
           <string>П</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rButton">
          <property name="text">
           <string>Р</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cButton">
          <property name="text">
           <string>С</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tButton">
          <property name="text">
           <string>Т</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="uButton">
          <property name="text">
           <string>У</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fButton">
          <property name="text">
           <string>Ф</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="xButton">
          <property name="text">
           <string>Х</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="czButton">
          <property name="text">
           <string>Ц</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chButton">
          <property name="text">
           <string>Ч</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shButton">
          <property name="text">
           <string>Ш</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="shchButton">
          <property name="text">
           <string>Щ</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="efButton">
          <property name="text">
           <string>Э</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yuButton">
          <property name="text">
           <string>Ю</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="yaButton">
          <property name="text">
           <string>Я</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string/>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Наименование</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="changePlant">
          <property name="text">
           <string>Посмотреть</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPlant">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePlant">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refreshPlant">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="searchLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchingButton">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Очистить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="newDB"/>
    <addaction name="exportDB"/>
    <addaction name="importDB"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newDB">
   <property name="text">
    <string>Новая база данных</string>
   </property>
  </action>
  <action name="exportDB">
   <property name="text">
    <string>Экспорт базы данных</string>
   </property>
  </action>
  <action name="importDB">
   <property name="text">
    <string>Импорт базы данных</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
